Gruntfile.js

```diff
@@ -34,154 +34,135 @@ module.exports = function (grunt) {
             },
 
             lintFiles: {
                 node: ['Gruntfile.js'],
                 es: ['<%= config.srcScriptsDir %>'],
-                tests: ['<%= config.testsDir %>']
-            }
-        }
+                tests: ['<%= config.testsDir %>'],
+            },
+        },
 
     });
 
     grunt.config.merge({
         watch: {
             es: {
                 files: [
-                    '<%= config.srcScriptsDir %>/**/*.es.js',
-                    '<%= config.compiledScriptsDir %>/**/templates.js'
+                    '<%= config.srcScriptsDir %>/**/*.js',
+                    '<%= config.compiledScriptsDir %>/**/templates.js',
                 ],
                 tasks: [
                     'compilejs',
-                    'uglify'
-                ]
+                    'uglify',
+                ],
             },
             swig: {
                 files: [
-                    '<%= config.templates.sourceFiles %>'
+                    '<%= config.templates.sourceFiles %>',
                 ],
                 tasks: [
-                    'orderedSwig'
-                ]
+                    'orderedSwig',
+                ],
             },
             ngtemplates: {
                 files: ['<%= config.templates.generatedFiles %>'],
-                tasks: ['ngtemplates']
-            }
-        },
-        browserify: {
-            all: {
-                files: {
-                    '<%= config.distDir %>/language-select.js': '<%= config.compiledScriptsDir %>/**/*.js'
-                }
-            }
+                tasks: ['ngtemplates'],
+            },
         },
+        browserify: {all: {files: {'<%= config.distDir %>/language-select.js': '<%= config.compiledScriptsDir %>/**/*.js'} } },
         babel: {
             all: {
-                options: {
-                    presets: ['es2015']
-                },
+                options: {presets: ['es2015']},
                 files: [{
                     expand: true,
                     cwd: '<%= config.srcScriptsDir %>',
                     src: ['**/*.es.js'],
                     dest: '<%= config.compiledScriptsDir %>',
-                    ext: '.js'
-                }]
-            }
+                    ext: '.js',
+                }],
+            },
         },
         eslint: {
+            options: {fix: grunt.option('fix-eslint')},
             node: {
-                options: {
-                    configFile: '.eslintrc.node'
-                },
-                src: '<%= config.lintFiles.node %>'
+                options: {configFile: '.eslintrc.node'},
+                src: '<%= config.lintFiles.node %>',
             },
             es: {
-                options: {
-                    configFile: '.eslintrc.es'
-                },
-                src: '<%= config.lintFiles.es %>'
+                options: {configFile: '.eslintrc.es'},
```

Comment:
Can you use eslint's "let me go find configs in parent folders and merge with them" auto config setting, instead of specifying files? The closer we can be to cli `eslint` working the same, the more we can make use of the linting: editors can use it, like Sublime.